unittests/test_clair/vulnerabilities-check_test.rego

```diff
@@ -29,3 +29,10 @@ test_warn_low_vulnerabilities {
     result[_].vulnerabilities_number == 1
     result[_].msg == "Found packages with low vulnerabilities. Consider updating to a newer version of those packages, they may no longer be affected by the reported CVEs."
 }
+
+test_warn_unknown_vulnerabilities {
+    result := warn_unknown_vulnerabilities with input as clair
+    result[_].details.name == "clair_unknown_vulnerabilities"
+    result[_].vulnerabilities_number == 1
+    result[_].msg == "Found packages with unknown vulnerabilities. Consider updating to a newer version of those packages, they may no longer be affected by the reported CVEs."
+}
```

Comment:
missing newline

Reply:
Ack, added

Reply:
ehm, I still don't see newline
![image](https://github.com/redhat-appstudio/hacbs-test/assets/20207783/c7729f5d-2a98-41fd-9a3f-5a40e11c1bf7)

Reply:
Doh, I somehow managed to add the newline to the wrong file in my hurry :smile: